book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
```

Comment:
- [ ] Here, in your book class, you are missing a method to add rentals to a book. Something similar to this: 
```
def add_rental (person, date)
 Rental.new(date, self, person)
end
```
Please make sure to add this method.